plugins/report/outline/lib.php

```diff
@@ -0,0 +1,131 @@
+<?php
```

Comment:
Looking at the docs [here](https://docs.moodle.org/dev/Reports), I see that some of the functions that define here are "magical" and result in navigation being exposed. However, it's not clear to me which ones we actually need if we're just wanting to expose a report for courses and teachers. Maybe that's understanding we'll have to solidify later, but I thought I'd ask if you experimented with any of that or may know from your work on this issue?